File: src/main/java/com/example/cineticket_pro/entities/Ingresso.java
package com.example.cineticket_pro.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor

public class Ingresso {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    public Long id;
    public String sessao;
    public int assento;
    public double valorPago;
    public EnumTipoIngresso status=EnumTipoIngresso.Inteira ;
    public LocalDate dataCompra;

}

File: front/app/(sistema)/types/ingresso.ts
export class Ingresso{
    constructor(
        public id: number | null,
        public sessao: String,
        public filme: String,
        public assento: number,
        public valorPago:number,
        public status: string,
        public statusIngresso: string,
        public dataCompra: string,


    ){}
}